examples/release_highlights/plot_release_highlights_0_22_0.py

```diff
@@ -19,6 +19,28 @@
     conda install scikit-learn
 """
 
+##############################################################################
+# KNN Based Imputation
+# ------------------------------------
+# We now support imputation for completing missing values using k-Nearest
+# Neighbors.
+#
+# Each sample’s missing values are imputed using the mean value from
+# ``n_neighbors`` nearest neighbors found in the training set. Two samples are
+# close if the features that neither is missing are close.
+# Please note that by default, a euclidean distance metric
+# that supports missing values,
+# :func:`~sklearn.metrics.nan_euclidean_distances`, is used to find the nearest
+# neighbors.
+#
+# Read more in the :ref:`User Guide <knnimpute>`.
+
+from sklearn.impute import KNNImputer
+nan = float("NaN")
```

Comment:
We don't want to use `np.nan` (just asking no strong feeling).

Reply:
That's how it is in the docstring, I just copied what was there. I have absolutely no preference here, other than probably having them consistent with each other.

Reply:
I should have just used `np.nan` in the `KNNImputer` docstring. I think more people know about `np.nan` than `float("NaN")`.

Reply:
we also use `np.nan` in the GBDT example